distributed/client.py

```diff
@@ -3035,22 +3035,11 @@ def restart(self, **kwargs):
         return self.sync(self._restart, **kwargs)
 
     async def _upload_file(self, filename, raise_on_error=True):
```

Comment:
It looks like `raise_on_error` isn't used anymore. Since we're maintaining our previous behavior (`test_upload_file_exception` and `test_upload_file_exception_sync` both pass still) this might be a good opportunity to remove `raise_on_error` as a keyword argument.